src/icons/__generated/Edit2.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/display-name */
+/* eslint-disable import/order */
+// DO NOT EDIT - GENERATED FILE
+/* exported and autogenerated using SVGR from https://www.figma.com/file/ZHiNW0Wf2bj2VHVIqpjMuh/Design-Library?node-id=495%3A629 */
+import { SvgIcon, SvgIconProps } from '../../ui/SvgIcon/SvgIcon';
```

Comment:
rename to Pencil ?

Reply:
i think this name comes from feather icons